### PR TITLE
feat(footer): add footer to layout

### PR DESCRIPTION
**Pull request by AaronCGoidel**
add footer element to every page through index.html
add styling for footer stuck to bottom of page

Closes: #21

### CODE REVIEW

**Comment by arnaud512** [comment]
Hey! thanks for the pr!
Can you add it in an angular component instead of the index.html?

File: src/index.html [hunk 1]
```diff
@@ -9,5 +9,12 @@
 </head>
 <body>
   <app-root></app-root>
+  <footer>
```
Comment (arnaud512) [inline src/index.html:12]: Can you move it to an Angular component?

File: src/styles.css [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+  position: relative;
+  min-height: 100%;
+}
+
 body {
     padding-top: 4.5rem;
+    padding-bottom: 4.5rem;
+}
+
+footer {
```
Comment (arnaud512) [inline src/styles.css:14]: and move this in that component?

**Review by arnaud512 — CHANGES_REQUESTED** [review]

**Comment by AaronCGoidel** [comment]
Will do. I'll put it in `src/app/footer`

**Review by arnaud512 — APPROVED** [review]

**Comment by arnaud512** [comment]
Hey @AaronCGoidel it's good for me! I'll merge it! thanks!

**Pull request merged**